### ISSUE TITLE
Test Query w/2 Shared Section Members - won't recognized that credentials are set

### ISSUE THREAD

**Issue by bgribaudo** [microsoft/vscode-powerquery-sdk#266]
### Preflight Checklist

- [X] I have installed the [latest version of Power Query SDK](https://github.com/microsoft/vscode-powerquery-sdk/releases/latest).
- [X] I have checked existing resources, including the [common issues](https://learn.microsoft.com/en-us/power-query/commonissues) and the [release notes](https://github.com/microsoft/vscode-powerquery-sdk/releases).
- [X] I have searched for [similar issues](https://github.com/microsoft/vscode-powerquery-sdk/issues).

### Power Query SDK

0.2.2

### Platform

Windows

### Architecture

x64

### Bug Description

Test query with multiple shared members can't be evaluated due to complaint about credentials needing to be set (even though they are set).

### Steps to Reproduce

1. Create a simple connector:
   ````
   section Test;

   [DataSource.Kind="Test", Publish="Test.Publish"]
   shared Test.Contents = () =>
       "hi";

   Test = [ Authentication = [Anonymous = []] ];

   Test.Publish = [
       Beta = true,
       Category = "Other",
       ButtonText = { "Title", "Help" }
   ];
   ````
2. Create a test query:
    ````
    section MyTester;

    shared A = Test.Contents();

    shared B = Test.Contents();
   ````
3. Build the connector (completes successfully)
4. Set credentials (completes successfully)
5. Attempt to evaluate the test query file.

### Actual Experience

````
[10:03:42 AM]	[Info]	[Task found] c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.2.2-win32-x64\.nuget\Microsoft.PowerQuery.SdkTools.2.114.4\tools\PQTest.exe run-test
				--extension "c:\Users\ben\Desktop\Test\bin\AnyCPU\Debug\Test.mez"
				--queryFile "c:\Users\ben\Desktop\Test\Test.query.pq"
				--prettyPrint
[10:03:45 AM]	[Error]	[Task exited abnormally] pqtest run-test pid(2728) exit(4294967291) stderr: Error: exception while running expression.: Credentials are required to connect to the Test source. (Source at Test.)
````

### Expected Experience

The first shared section member in the test query should run successfully.

### Additional Context

Interestingly, the set credentials action does not have a problem with the presence of the two shared section members. 

Real life scenario for multiple shared section members in test query: Test query contains unit tests but then a need arises for manual testing. Manual test code is placed in a separate section member above the unit test-containing section member so that it stays untouched.

**Comment by albertli-msft** [comment]
![image](https://github.com/microsoft/vscode-powerquery-sdk/assets/69623692/0f6b6897-5c2c-4601-b4bb-42a6995f62db)
![image](https://github.com/microsoft/vscode-powerquery-sdk/assets/69623692/177db476-387b-4adb-a0ee-451d5bd1dff4)

This would be fixed in our SdkTool of 2.117.* 😀

**Comment by mattmasson** [comment]
This should be resolved in the [0.3.1 release](https://github.com/microsoft/vscode-powerquery-sdk/releases/tag/v0.3.1) (sdktools `2.127.2`).

**Comment by bgribaudo** [comment]
Thank you, @mattmasson!
